app/templates/components/events/event-import-section.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="ui stackable centered grid">
+  <div class="sixteen wide column">
+    <h3 class="ui header">{{t 'Import Event'}}</h3>
+    <div class="ui divider"></div>
+    <p class="section-heading">{{t 'Select event source file to import'}}</p>
+    <p><b>{{t 'Supported formats:'}}</b> {{t 'Open Event compatible json package (.zip), PentabarfXML (.xml), iCalendar (.ical)'}}</p>
+    {{input type='file'}}
```

Comment:
Let's re-arrange this a bit for uniformity. 

```hbs
<div class="field">
    <label class='required'> Select event source file to import</label> <!-- for uniform labels across the app -->
    {{input type='file'}}
    <br> <!-- this is how help text is displayed across the app (Help text below input). Lets follow the same pattern so as to not confuse the user -->
    <b>{{t 'Supported formats:'}}</b> {{t 'Open Event compatible json package (.zip), PentabarfXML (.xml), iCalendar (.ical)'}}
</div>
```

And put all of this and the button inside a form. And add `type="submit"` to the button.